model/users.go

```diff
@@ -0,0 +1,30 @@
+package model
+
+import "time"
+
+type User struct {
+	UserID         int       `json:"user_id"`
+	GithubUsername string    `json:"github_username"`
+	GithubID       int       `json:"github_id"`
```

Comment:
The convention for JSON is `camelCase`, so let us obey that:

```suggestion
	GithubUsername string    `json:"githubUsername"`
	GithubID       int       `json:"githubId"`
```